packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -439,6 +439,28 @@ - (void)testSeekToleranceWhenSeekingToEnd {
   XCTAssertNil(error);
   XCTAssertEqual(avPlayer.volume, 0.1f);
 
+  // Set picture-in-picture
```

Comment:
I'm not sure why the existing test is written this way, but we should have one test method per logical test. So all of this should be in new test methods that each exercise specific parts of the new logic.

They should also fully cover it, which means at least one call to each method, and probably more (for branches).